internal/gcs/process.go

```diff
@@ -104,28 +106,39 @@ func (gc *GuestConnection) exec(ctx context.Context, cid string, params interfac
 	p.id = resp.ProcessID
 	// Start a wait message.
 	waitReq := containerWaitForProcess{
-		requestBase: makeRequest(cid),
+		requestBase: makeRequest(ctx, cid),
 		ProcessID:   p.id,
 		TimeoutInMs: 0xffffffff,
 	}
 	p.waitCall, err = gc.brdg.AsyncRPC(ctx, rpcWaitForProcess, &waitReq, &p.waitResp)
 	if err != nil {
 		return nil, fmt.Errorf("failed to wait on process, leaking process: %s", err)
 	}
+	go p.waitBackground()
 	return p, nil
 }
 
 // Close releases resources associated with the process and closes the
 // associated standard IO streams.
 func (p *Process) Close() error {
+	_, span := trace.StartSpan(context.Background(), "gcs::Process::Close")
+	defer span.End()
+	span.AddAttributes(
+		trace.StringAttribute("cid", p.cid),
+		trace.Int64Attribute("pid", int64(p.id)))
+
 	p.stdin.Close()
```

Comment:
I notice we don't return any errors here, but how would you feel about logging (log.G) if we hit errors closing any of these?

Reply:
Done.